src/Tribe/Aggregator/Event.php

```diff
@@ -349,27 +353,27 @@ public static function preserve_changed_fields( $data ) {
 				continue;
 			}
 
-			// if we don't have a field to reset to, let's unset the event meta field
-			if ( ! isset( $post_meta[ $field ] ) ) {
-				unset( $data[ $field ] );
```

Comment:
I mean here:  https://github.com/moderntribe/the-events-calendar/pull/1353/files#diff-f8677ebdc85d2610cfcd992b86b08790R364